statistical/calculate_medians_LI.py

```diff
@@ -0,0 +1,131 @@
+"""Calculate medians using linear interpolation"""
+import json
+from scipy import stats
+import numpy as np
+import pandas as pd
+
+z_score_90 = stats.norm.ppf(0.9)
```

Comment:
wonder why this was changed to 90 again. Since the only place I am seeing it being used is the cv calculation. I thought we settled the .95 is the correct one to use.

Reply:
ok went back to Erica's message and see that 1.645 is correct z-score to use, pushing this change now